server/sonar-server/src/main/java/org/sonar/server/permission/index/AuthorizationDao.java

```diff
@@ -35,7 +35,7 @@
 /**
  * No streaming because of union of joins -> no need to use ResultSetIterator
  */
-public class IssueAuthorizationDao {
+public class AuthorizationDao {
```

Comment:
As there's already a class AuthorizationDao, I suggest to rename this one PermissionIndexerDao. The term "authorization" is used for verification of permissions for now.